languages/typescript/src/main/kotlin/io/vrap/codegen/languages/typescript/joi/JoiValidatorModuleRenderer.kt

```diff
@@ -120,7 +120,7 @@ class JoiValidatorModuleRenderer constructor(override val vrapTypeProvider: Vrap
         val patternProperties = this.allProperties
                 .filter { it.isPatternProperty() }.joinToString(separator = "\n") { ".pattern(${it.asRegExp()}, ${it.type.toVrapType().simpleJoiName()}())" }
         val additionalProperties = this.additionalProperties?:true
-        val unknown = if (additionalProperties) ".unknown()" else ""
+        val unknown = if (additionalProperties) ".unknown(false)" else ""
```

Comment:
```suggestion
        val unknown = if (additionalProperties) ".unknown()" else ".unknown(false)"
```

Reply:
> we could be able to do it by adding flag `additionalProperties` as `false` , ref [raml doc](https://github.com/raml-org/raml-spec/blob/master/versions/raml-10/raml-10.md?rgh-link-date=2021-02-19T18%3A47%3A02Z#object-type)
> like here [raml spec import api](https://github.com/commercetools/commercetools-importer/blob/master/api-spec/types/product-import.raml?rgh-link-date=2021-02-19T18%3A47%3A02Z#L64)

So it would be more like this.

Reply:
But still I would go the way to have some kind of JoiValidator configuration to toggle it.